addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/antlr/README.md

```diff
@@ -0,0 +1,13 @@
+## JavaScript Lexer
+
+The JavaScript lexer used in this add-on is generated using ANTLR with their [JavaScript grammar](https://github.com/antlr/grammars-v4/blob/ff2f66fa8663e52a41b8ff2ad7d6237e4b3293d4/javascript/javascript/JavaScriptLexer.g4).
+
+The following files were copied from the referenced repository:
+ - [`JavaScriptLexerBase.java`](src/main/java/org/zaproxy/zap/extension/pscanrules/antlr/JavaScriptLexerBase.java);
+ - [`JavaScriptLexer.g4`](src/main/antlr/org/zaproxy/zap/extension/pscanrules/antlr/JavaScriptLexer.g4);
+
+The lexer is automatically generated when the code is compiled through the [`antlr` Gradle plugin](https://docs.gradle.org/current/userguide/antlr_plugin.html).
+
+If/when the JavaScriptLexer.g4 is updated:
+- JavaScriptLexerBase.java may also need to be updated/replaced. (You'll need to add the appropriate package declaration.)
+- Run: `./gradlew :aO:pscanrules:generateGrammarSource` to update/recreate JavaScriptLexer.java.
```

Comment:
Are these really needed? The second sure not, the class is generated automatically. I'd avoid adding unnecessary comments (we'll also automate the copy/update from upstream).

Reply:
I guess I could drop the file and just prioritize setting up the automation as a short term task?

It should be pretty small/quick and removes the need for this info completely (IMHO).